### PR TITLE
Update translation PT-BR

### PR DESCRIPTION
**Pull request by duhnunes**
• Updated according to original
• Minor changes in translation

### CODE REVIEW

**Comment by mchorse** [comment]
Hey, sorry for not replying, but at the moment, I'm rewriting a lot of stuff in Aperture without caring for translation strings yet. If you want, you can follow along as I update stuff, but there will be more changes until I finish fully the update, so it's a better idea to wait a bit. 😅

**Comment by duhnunes** [comment]
> Hey, sorry for not replying, but at the moment, I'm rewriting a lot of stuff in Aperture without caring for translation strings yet. If you want, you can follow along as I update stuff, but there will be more changes until I finish fully the update, so it's a better idea to wait a bit. 😅

Haha, no problem, that's what I thought. I come here from time to time and keep updating the files.
Don't worry, I just like to keep updated and not accumulate much work for later, so I always make changes, even if they are not permanent in the final product. I also take the opportunity and make some revisions and if necessary, make modifications.
In particular, writing is a therapy to me, so I don't care if it is necessary to erase something I have already done.
🙂

File: help/pt_BR/config.yml [hunk 1]
```diff
@@ -1,61 +1,74 @@
...
-        roll_friction: Atrito de rolagem
-
+        roll_friction: Deslizamento de rolagem
         fov_speed: Velocidade do FOV
-        fov_friction: Atrito do FOV
-
-        smooth_enabled: Câmera suave ativada
-        mouse_x_friction: Atrito do mouse X
-        mouse_y_friction: Atrito do mouse Y
+        fov_friction: Deslizamento do FOV
```
Comment (mchorse) [inline help/pt_BR/config.yml:76]: You missed here `minema` [section](https://github.com/mchorse/aperture/compare/1.3.6...1.12#diff-7c88845ae8f8e0f54123ce9e3dfaf823R76).

File: help/pt_BR/config_comments.yml [hunk 2]
```diff
@@ -1,47 +1,57 @@
...
-        mouse_x_friction: Atrito suave do mouse X
-        mouse_y_friction: Atrito suave do mouse Y
+        enabled: Ativar câmera suave
+        x_friction: Deslizamento suave do mouse em X
+        y_friction: Deslizamento suave do mouse em Y
+        clamp: Prender a câmera numa inclinação suave entre -90 e 90 graus de intervalo?
+        roll_speed: Velocidade de aceleração de rolagem
+        roll_friction: Deslizamento de aceleração de rolagem (com que rapidez diminui)
+        fov_speed: Velocidade de aceleração do FOV
+        fov_friction: Delizamento de aceleração do FOV (com que rapidez diminui)
```
Comment (mchorse) [inline help/pt_BR/config_comments.yml:59]: Here `minema` section [is missing](https://github.com/mchorse/aperture/compare/1.3.6...1.12#diff-01caf0c8a890718ab1c240aed7357a4aR59) too.

**Comment by mchorse** [comment]
Alright, I checked everything, it seems only `minema` sections are missing. Add those, and I'll accept pull request. I plan to publish the 1.4 update today in an hour, however, once translations will come, I'll release a patch with language strings.

> Haha, no problem, that's what I thought. I come here from time to time and keep updating the files.
> Don't worry, I just like to keep updated and not accumulate much work for later, so I always make changes, even if they are not permanent in the final product. I also take the opportunity and make some revisions and if necessary, make modifications.
> In particular, writing is a therapy to me, so I don't care if it is necessary to erase something I have already done.

I understand now, I really appreciate the translations. Thank you! 😊

**Comment by duhnunes** [comment]
Hello,

Sorry for the delay. I've been out of free time lately.

I compared the files edited by you, on the dates I was offline and did the file comparisons, adding and changing things, I believe everything is in order now.

I will be more willing during those days, let me know if there is anything that needs to be fixed.

I hope you are well, take care! Big hug! (:

**Comment by Chryfi** [comment]
Thanks for your translations. And sorry for the delay; McHorse quit, and I didn't know about these translation pull requests. I will go through them when Aperture 1.8 update comes out in maybe some weeks hopefully.

**Pull request closed without merging**